packages/terra-table/docs/themeable-variables.md

```diff
@@ -0,0 +1,3 @@
+# Themeable Variables
+
```

Comment:
Added variables in doc https://github.com/cerner/terra-core/pull/177/commits/c9b2d6834df2d0749688c1158f19abec55eb5a0e#diff-a2f8fe2900b7404b7406ee8301ba4387R4